pkgs/tools/package-management/micromamba/default.nix

```diff
@@ -66,6 +66,7 @@ stdenv.mkDerivation rec {
     description = "Reimplementation of the conda package manager";
     homepage = "https://github.com/mamba-org/mamba";
     license = licenses.bsd3;
+    platforms = platforms.all;
```

Comment:
This bit has just been merged to master https://github.com/NixOS/nixpkgs/pull/178801/commits/fcda800ae2a95fb10c432ce715030d6a6e8fd402
so let's remove it from this PR.